### DISCUSSION TITLE
[2022/09/27] Frontend Night

### DISCUSSION THREAD

**Discussion by jjenko** [wantedly/frontend-night#224] [Archive]
## Policy

- 資料の準備をがんばらない
- 参加・途中退席は自由
- スキップしない

話したい・聞きたいネタを書いてく ✏️

ハッシュタグ [#frontend_night](https://twitter.com/hashtag/frontend_night?src=hash)

Prev: https://github.com/wantedly/frontend-night/discussions/223 

 _Created from https://github.com/wantedly/frontend-night/discussions/179 by [issue-creator](https://github.com/rerost/issue-creator)_

**Comment by chloe463** [comment]
- [TechFeed Expert Night #4 〜 フロントエンドアーキテクチャを語る](https://youtu.be/-6yy1VTcK-w?t=1751)
	- Astro, Qwik とかの話
- https://youtu.be/PSzCtdM20Fc
